src/main/scala/it/pagopa/interop/tenantprocess/service/impl/TenantManagementServiceImpl.scala

```diff
@@ -108,6 +108,15 @@ final case class TenantManagementServiceImpl(
       .toFuture(TenantAttributeNotFound(tenantId, attributeId))
   } yield attribute
 
+  def getTenantBySelfcareId(selfcareId: UUID)(implicit contexts: Seq[(String, String)]): Future[Tenant] = withHeaders {
+    (bearerToken, correlationId, ip) =>
+      val request =
+        tenantApi.getTenantBySelfcareId(xCorrelationId = correlationId, selfcareId = selfcareId, xForwardedFor = ip)(
```

Comment:
I'm probably missing something, but I would expect this request to be done in the read model.

Reply:
I am not able to find selfcareId to tenants collections

Reply:
The `selfcareId` is an optional field and is populated the first time an institution enters the FE.
We should return `404` when a `selfcareId` request fetches no tenant.